org.kde.kpat.json

```diff
@@ -0,0 +1,115 @@
+{
+    "id": "org.kde.kpat",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.13",
+    "sdk": "org.kde.Sdk",
+    "command": "kpat",
+    "finish-args": ["--share=ipc", "--socket=x11", "--socket=wayland" ],
```

Comment:
Nitpick: put a finish-arg per line.

Reply:
It's a useless change and increases the diff with the nightly .json file we have in KDE CI, so i'm not doing this

Reply:
or just fix the manifest upstream? it's just harder to read this way.

Reply:
Or we could leave it like this and save time discussing over what is more or less readable :)